src/utils/operator.ts

```diff
@@ -153,13 +153,17 @@ export function getOrderExecutedAmounts(order: RawOrder): {
   }
 }
 
-export type GetOrderPriceParams = {
+export type GetRawOrderPriceParams = {
   order: RawOrder
   buyTokenDecimals: number
   sellTokenDecimals: number
   inverted?: boolean
 }
 
+export interface GetOrderPriceParams extends Omit<GetRawOrderPriceParams, 'order'> {
```

Comment:
I had to add an extension to the property in `getOrderLimitPrice` to be able to us in [`OrdersUsersDetailsTable`](https://github.com/gnosis/gp-ui/blob/1044-orders-userdetails-table/src/components/orders/OrdersUserDetailsTable/index.tsx#L45). 

@alfetopito maybe we could use here only what is used inside which is `amount`, `token` and not the whole order.

Reply:
That suggestion makes a lot of sense.
Feel free to update the function to expect only those parameters instead of the whole order object.

As far as I remember it won't break anything as the function is not currently in use.

Also, now I remember why I wrote that function. 
I intended to use it when building the order object and have the limit price available (and execution price if there was a trade) as part of the Order object.
I ended up not using it because based only on the Order data is not possible to calculate the price, since we need both token decimals which are fetched on another call.